content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -329,7 +329,9 @@ message NUMANode {
 ```
 Device Plugins that wish to leverage the Topology Manager can send back a populated TopologyInfo struct as part of the device registration, along with the device IDs and the health of the device. The device manager will then use this information to consult with the Topology Manager and make resource assignment decisions.
 
-`TopologyInfo` supports a `nodes` field that is either `nil` (the default) or a list of NUMA nodes. This lets the Device Plugin publish that can span NUMA nodes.
+`TopologyInfo` has a `nodes` field that is either `nil` or a list of NUMA nodes. This lets the Device Plugin publish resources that can span NUMA nodes.
+
+`nil` `TopologyInfo` or an empty list of NUMA nodes indicates that the Device Plugin does not have NUMA affinity preference for the resource it manages.
```

Comment:
```suggestion
`TopologyInfo` supports setting a `nodes` field to either `nil` or a list of NUMA nodes. This lets the Device Plugin publish resources that can span multiple NUMA nodes.

Setting `TopologyInfo` to `nil`  or providing an empty list of NUMA nodes for a given device indicates that the Device Plugin does not have a NUMA affinity preference for that device.
```

Reply:
I wanted to avoid `device` because there might not be a _device_  the plugin manages (like in our case SGX that triggered this issue).

Reply:
I'd rather be consistent with the language of the rest of this section than switch to using `resource` here (even if all of the resources advertised by the device plugin are not always strictly devices).

Reply:
I pushed these suggestions. The test of the doc seems to talk about _resource_ in the same context so it looks only this section was written using "_device_".